aws.yaml

```diff
@@ -34,10 +34,10 @@ inputs:
     default: { get_attribute: [ kubernetes_master_ip, aws_resource_id ] }
 
   ks_node_dep_archive:
-    default: https://github.com/cloudify-incubator/kubernetes-node-blueprints/archive/master.zip
+    default: https://codeload.github.com/cloudify-incubator/kubernetes-node-blueprints/zip/master
 
   ks_load_dep_archive:
-    default: https://github.com/cloudify-incubator/kubernetes-lb-blueprints/archive/master.zip
+    default: https://codeload.github.com/cloudify-incubator/kubernetes-lb-blueprints/zip/master
 
```

Comment:
What do you think that we move this input into `imports/kubernetes.yaml` so that we don't have it duplicated in every bp?

Reply:
Yeah sure will update that and totally agree